Microsoft.ML.sln

```diff
@@ -15,8 +15,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ML.Core.Tests", "
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ML.CpuMath", "src\Microsoft.ML.CpuMath\Microsoft.ML.CpuMath.csproj", "{46F2F967-C23F-4076-858D-33F7DA9BD2DA}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ML.PipelineInference", "src\Microsoft.ML.PipelineInference\Microsoft.ML.PipelineInference.csproj", "{2D7391C9-8254-4B8F-BF26-FADAF8F02F44}"
```

Comment:
Doing this will make it not build anymore. Is that what we intend?  If yes, we may as well delete the code IMO. Someone can get it from source control history if they need it.

Reply:
Yes, InferencePipeline needs to be removed. I can build my changes locally. Do you mean that CI will still crash?

Reply:
I think "will not build" is meant literally, as in, it will not build, not it cannot build. But of course either way removing it is as @eerhardt suggests is the right thing to do.

---
In reply to: [235221248](https://github.com/dotnet/machinelearning/pull/1686#discussion_r235221248) [](ancestors = 235221248)

Reply:
It's removed in my PR now.

---
In reply to: [235445964](https://github.com/dotnet/machinelearning/pull/1686#discussion_r235445964) [](ancestors = 235445964,235221248)